components/match2/wikis/splitgate/match_group_input_custom.lua

```diff
@@ -0,0 +1,553 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local TypeUtil = require('Module:TypeUtil')
```

Comment:
you can kick this and use module logic instead since the only function used from this module exists in the logic module too